tests/manage/ocs_288.py

```diff
@@ -0,0 +1,407 @@
+    assert validate_pool_creation(pool_name)
+    if validate_pool_creation(pool_name):
+        log.info("Resource validation Passed")
+    else:
+        return 1
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        SC_RBD_YAML, **kwargs
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a new RBD StorageClass")
+    assert SC.create(yaml_file=TEMP_YAML_FILE)
+    assert validate_storageclass(sc_name)
+    return True
+
+
+def get_pool_get_sc_name():
+    """
+    This will generate random pool_name and storageclass name
+    :return:
+    """
+    sc_data = {}
+    pool_name = "pool-" + ''.join(
+        [random.choice(string.ascii_lowercase) for _ in range(4)])
+    sc_name = "ocsci-csi-" + pool_name
+    sc_data['rbd_storageclass_name'] = sc_name
+    sc_data['rbd_pool'] = pool_name
+    return sc_data
+
+
+def get_storageclass():
+    """
+    Funtion for getting all storageclass
+    :return: storageclass list
+    """
+    sc_obj = SC.get()
+    sample = sc_obj['items']
+    ignore_sc = 'gp2'
+    for item in sample:
+        if item.metadata.name not in ignore_sc:
+            storageclass_list.append(item.metadata.name)
+    if len(storageclass_list) >= 1:
+        return True
+    else:
+        log.error("No StorageClass Found")
+        return False
+
+
+def create_secret(file_name):
+    """
+    This will create Secret file which will be used for creating StorageClass
+    :return:
+    """
+    secret_data = {}
+    user = "admin"
+    admin_byte = base64.b64encode(user.encode("utf-8"))
+    admin_base64 = str(admin_byte, "utf-8")
+    cmd = f"ceph auth get-key client.admin"
+    out = ocp.exec_ceph_cmd(cmd)
+    admin_key = out.get('key')
+    admin_key_byte = base64.b64encode(admin_key.encode("utf-8"))
+    admin_key_base64 = str(admin_key_byte, "utf-8")
+    secret_data['base64_encoded_admin_password'] = admin_key_base64
+    secret_data['base64_encoded_admin'] = admin_base64
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        file_name, **secret_data
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    assert SECRET.create(yaml_file=TEMP_YAML_FILE)
+    secret_list.append(file_y.get('metadata')['name'])
+    return True
+
+
+def create_pvc(**kwargs):
+    """
+    Function for creating pvc and multiple pvc
+    :param kwargs:
+    :return:
+    """
+    if kwargs.get('count'):
+        pvc_count = int(kwargs.get('count'))
+    else:
+        pvc_count = 1
+    for i in range(pvc_count):
+        sc_name = random.choice(storageclass_list)
+        pvc_data = {}
+        pvc_data['rbd_storageclass_name'] = sc_name
+        pvc_data['pvc_name'] = "rbd-pvc-" + ''.join(
+            [random.choice(string.ascii_lowercase)
+             for _ in range(3)]) + str(sc_name)
```

Comment:
please add explanation of what you are doing here.
I think we already have a function to do that

Reply:
Here i am creating pvc with prefix rbd-pvc and appending 3 random letter and at suffix appending storageclass name
Example:-
1:- rbd-pvc-eaccsi-cephfs (here eac is random text and csi-cephfs is storageclass)
2:- rbd-pvc-fwqocsci-csi-pool-ejtn (here fwq is random text and ocsci-csi-pool-ejtn is storageclass name)